effectsize.Rmd

```diff
@@ -0,0 +1,99 @@
+# Effect size {#effectsize}
+<div style="text-align:center">
+```{r}
+knitr::include_graphics("figures/effectsize/teaser.png")
```

Comment:
I have used gtable for this before: https://github.com/baptiste/gridextra/wiki/gtable

Although in this case I think you may not need it, and that it would probably complicate life rather than simplify it. My guess is that you could recreate the plot you have by laying out the geoms necessary within a single plot.